test/bin/mdx-test/expect/simple-mli/test-case.mli

```diff
@@ -19,14 +19,30 @@
       print_endline (String.concat " " [the; last; phrase])
     ]}
 
-    {[
+    With the optional header:
+
+    {@ocaml[
       # List.map (fun x -> x * x) [(1 + 9); 2; 3]
       - : int list = [100; 4; 9]
       # List.map (fun x -> x * x) [1; 2; 3]
       - : int list = [1; 4; 9]
     ]}
+
+    A shell block:
+
+    {@sh set-FOO=Hello,set-BAR=Bash[$ echo $FOO $BAR
+Hello Bash
+
+    ]}
```

Comment:
The initial newline character is moved to the end in this `sh` block. Any idea ?